test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-ui-state.json

```diff
@@ -287,6 +287,7 @@
     "queuedRequestCount": 0,
     "fcmToken": "string",
     "remoteFeatureFlags": {},
+    "requests": "object",
```

Comment:
I don't think this was surfaced to the UI before and it also really doesn't need to be?

Reply:
Any idea how to remove it? I haven't changed anything related to it AFAIK.